avd_docs/azure/container/AVD-AZU-0041/Terraform.md

```diff
@@ -3,9 +3,13 @@ Limit the access to the API server to a limited IP range
 
 ```hcl
  resource "azurerm_kubernetes_cluster" "good_example" {
-     api_server_authorized_ip_ranges = [
+	api_server_access_profile {
+		authorized_ip_ranges = [
  		"1.2.3.4/32"
  	]
+
+	}
+     
  }
  
```

Comment:
ditto

Reply:
Same appear as the result of make docs command.

Reply:
Same appear as the result of make docs command.